$OpenBSD: patch-includes_defaults_inc_php,v 1.31 2018/02/12 23:27:18 sthen Exp $

Some substitutions are done in the ports Makefile to reduce patch conflicts
during updates.

Index: includes/defaults.inc.php
--- includes/defaults.inc.php.orig
+++ includes/defaults.inc.php
@@ -42,14 +42,15 @@ $config['db_socket']             = null;
 $config['own_hostname'] = 'localhost';
 
 // Location of executables
-//$config['fping']                    = '/usr/sbin/fping';
-//$config['fping6']                   = '/usr/sbin/fping6';
+$config['fping']                    = '/usr/local/sbin/fping';
+$config['fping6']                   = '/usr/local/sbin/fping6';
 // https://docs.librenms.org/Support/Configuration/#fping
 $config['fping_options']['timeout'] = 500;
 $config['fping_options']['count']   = 3;
 $config['fping_options']['interval'] = 500;
 $config['snmpwalk']                  = '/usr/local/bin/snmpwalk';
 $config['snmpget']                   = '/usr/local/bin/snmpget';
+$config['snmpgetnext']               = '/usr/local/bin/snmpgetnext';
 $config['snmpbulkwalk']              = '/usr/local/bin/snmpbulkwalk';
 $config['snmptranslate']             = '/usr/local/bin/snmptranslate';
 $config['whois']          = '/usr/bin/whois';
@@ -495,11 +496,11 @@ $config['billing']['bill_autoadd'] = 0;
 $config['billing']['base'] = 1000;
 // Set the base to divider bytes to kB, MB, GB ,... (1000|1024)
 // External Integration
-// $config['rancid_configs'][]             = '/var/lib/rancid/network/configs/';
+// $config['rancid_configs'][]             = '/var/rancid/network/configs/';
 $config['rancid_ignorecomments'] = 0;
 // Ignore lines starting with #
-// $config['collectd_dir']                 = '/var/lib/collectd/rrd';
-// $config['smokeping']['dir']             = "/var/lib/smokeping/";
+// $config['collectd_dir']                 = '/var/collectd/rrd';
+// $config['smokeping']['dir']             = "/var/db/smokeping/";
 $config['smokeping']['pings']             = 20;
 // $config['oxidized']['enabled']         = FALSE;//Set to TRUE
 // $config['oxidized']['url']             = 'http://127.0.0.1:8888';// Set the Oxidized rest URL
@@ -507,7 +508,7 @@ $config['smokeping']['pings']             = 20;
 // NFSen RRD dir.
 $config['nfsen_enable'] = 0;
 // $config['nfsen_split_char']   = "_";
-// $config['nfsen_rrds']   = "/var/nfsen/profiles-stat/live/";
+// $config['nfsen_rrds']   = "/var/db/nfsen/profiles-stat/live/";
 // $config['nfsen_suffix']   = "_yourdomain_com";
 // Location Mapping
 // Use this feature to map ugly locations to pretty locations
@@ -853,7 +854,7 @@ $config['api_demo'] = 0;
 // Set this to 1 if you want to disable some untrusting features for the API
 // Distributed Poller-Settings
 $config['distributed_poller']                = false;
-$config['distributed_poller_name']           = trim(file_get_contents('/proc/sys/kernel/hostname'));
+$config['distributed_poller_name']           = trim(file_get_contents('/etc/myname'));
 $config['distributed_poller_group']          = 0;
 $config['distributed_poller_memcached_host'] = 'example.net';
 $config['distributed_poller_memcached_port'] = '11211';
